cabal.GHC-8.8.4.config

```diff
@@ -147,7 +148,7 @@ constraints: any.Cabal ==3.0.1.0,
              any.gitrev ==1.3.1,
              any.happy ==1.20.0,
              any.hashable ==1.3.1.0,
-             hashable +integer-gmp,
+             hashable -examples +integer-gmp +sse2 -sse41,
```

Comment:
What's the motivation for this change? Is it consistent in the other freeze files?

Reply:
Also no idea what this one is about either. As the previous one, I would guess this is an artifact of past merges...?

Reply:
If we don't know why these flags were turned on, then we can probably just turn them back off